quartz/src/main/java/org/quartz/impl/jdbcjobstore/Util.java

```diff
@@ -179,6 +181,59 @@ private static java.lang.reflect.Method getSetMethod(String name, PropertyDescri
         return null;
     }
 
+    static boolean containsColumnNames(ResultSet rs, String... colNames) throws SQLException {
+        ResultSetMetaData rsmd = rs.getMetaData();
+        int columnCount = rsmd.getColumnCount();
+
+        for (int i = 1; i <= columnCount; i++ ) {
+            String name = rsmd.getColumnName(i);
+            for (String colName : colNames) {
+                if (colName.equals(name)) {
```

Comment:
Are you sure this should not be checked case-insensitive? For example, the Postgres driver has lowercase column names in the result set metadata. The arguments passed to this method are in uppercase.

Reply:
Just to be clear, this is a regression in 2.5.1 which blocks upgrading to this version